Sources/SwiftSyntax/Raw/gyb_generated/RawSyntaxNodes.swift

```diff
@@ -8531,6 +8680,7 @@ public struct RawAccessorDeclSyntax: RawDeclSyntaxNodeProtocol {
     _ unexpectedAfterBody: RawUnexpectedNodesSyntax? = nil,
     arena: __shared SyntaxArena
   ) {
+    assert(accessorKind.tokenKind == .keyword(.get) || accessorKind.tokenKind == .keyword(.set) || accessorKind.tokenKind == .keyword(.didSet) || accessorKind.tokenKind == .keyword(.willSet) || accessorKind.tokenKind == .keyword(.unsafeAddress) || accessorKind.tokenKind == .keyword(.addressWithOwner) || accessorKind.tokenKind == .keyword(.addressWithNativeOwner) || accessorKind.tokenKind == .keyword(.unsafeMutableAddress) || accessorKind.tokenKind == .keyword(.mutableAddressWithOwner) || accessorKind.tokenKind == .keyword(.mutableAddressWithNativeOwner) || accessorKind.tokenKind == .keyword(._read) || accessorKind.tokenKind == .keyword(._modify), "Received \(accessorKind.tokenKind)")
```

Comment:
These should overlap with the subset right? Wonder if we should autogenerate subsets when there's > 1 kind. I imagine that would be a lot though, maybe only > 3 and then `at` can have overloads for up to 3 kinds or a subset?

Reply:
That’s the next item on my To-Do list after this PR is merge: “Generate RawTokenKindSubset”. I’m still not entirely sure how exactly we want to decide whether to generate the `RawTokenKindSubset`, but it’s definitely something we should do.